yarn.lock

```diff
@@ -609,7 +618,7 @@ css-what@2.1:
   version "2.1.0"
   resolved "https://registry.yarnpkg.com/css-what/-/css-what-2.1.0.tgz#9467d032c38cfaefb9f2d79501253062f87fa1bd"
 
-css@2.2.1, css@^2.1.0, css@^2.2.0, "css@https://github.com/pocketjoso/css.git":
+css@2.2.1, css@^2.1.0, css@^2.2.0:
   version "2.2.1"
   resolved "https://github.com/pocketjoso/css.git#8ddea7e3cbc0a183ecf694a7a5fbc84326893893"
```

Comment:
It's still present here.. I find it quite annoying to upgrade sub-dependencies with yarn... Normally enough to do: `yarn remove penthouse && yarn add penthouse`. (however will only work if penthouse is the only dep on this fork of css, which should be the case though)

Reply:
I think fixing this will fix the build problem.

Reply:
Thank you for the review and catching this! I tried "reyarning" but it didn't remove it. Seems some other packages also need it.

```
S/l/critical (fix-github-package) $ yarn why css                                          20s 806ms
yarn why v0.27.5
[1/4] Why do we have the module "css"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
info Reasons this module exists
   - "filter-css" depends on it
   - "inline-critical" depends on it
   - "inline-critical#cave" depends on it
```

Reply:
yeah this is what I was fearing. I don't have any better method for this than to yarn remove all of them, re-adding all of them...

You could also go in and just paste the correct resolve url into the yarn file, but generally that's yucky.. [this](https://github.com/pocketjoso/penthouse/blob/master/yarn.lock#L998) should be the right line though.